### PR TITLE
Connect to be v2

### PR DESCRIPTION
**Pull request by attom2**
# Pull request checklist

## Pull request type

Please check the type of change your PR introduces:
image

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Current behavior

- No calls to backend

## New behavior

- Calls to backend for grabbing initial Campaigns, Post request to backend which creates a new campaign

### CODE REVIEW

File: src/components/CampaignCard.vue [hunk 1]
```diff
@@ -6,9 +6,9 @@
     <div>
       <section class="campaign-card">
         <section>
-        <div class="img-div">Image goes here</div>
+        <div class="img-div"><img :src=campaign.image /></div>
```
Comment (attom2) [inline src/components/CampaignCard.vue:9]: easy way to reference image given by backend

File: src/components/CampaignDetails.vue [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
-    <h3> Minimum Contribution </h3>
-    <h1>{{ campaign.minContribution }}</h1>
-    <p>There is some information missing and this has no styles </p>
-    <input v-model='contribution' placeholder='Enter your contribution' type="number"/>
-    <button :disabled="!contribution"
-    @click="submitContribution">
-    Submit Contribution </button>
-
-  </section>
+  <div class="details-card">
```
Comment (attom2) [inline src/components/CampaignDetails.vue:3]: cleaned up card details, Lots more could be done in terms of styling

File: src/components/CampaignDetails.vue [hunk 3]
```diff
@@ -29,7 +33,7 @@ export default {
   props: ['address'],
   computed: {
     campaign() {
-      return this.$store.getters.getSingleCampaign(Number(this.address))
+      return this.$store.getters.getSingleCampaign(this.address)
```
Comment (attom2) [inline src/components/CampaignDetails.vue:36]: addresses are no longer numbers, now real ethereum addresses

**Review by jpc20 — APPROVED** [review]
Looks great, love the mullet fund.

**Review by edmdc — APPROVED** [review]

**Pull request merged**